### PR TITLE
MWPW-142200 ability to add strikethrough fro price for Special offer

### PR DESCRIPTION
**Pull request by VKniaz**
Resolves: [MWPW-142200](https://jira.corp.adobe.com/browse/MWPW-142200)

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/drafts/vkniaziev/document6?martech=off
- After: https://mwpw-142200--milo--adobecom.hlx.page/drafts/vkniaziev/document6?martech=off

### CODE REVIEW

**Review by Axelcureno — APPROVED** [review]

File: libs/blocks/merch-card/merch-card.css [hunk 1]
```diff
@@ -9,11 +9,15 @@ div[class*="-merch-card"] > div,
...
+  margin-top: 8px;
+  margin-bottom: 16px;
 }
 
 .dark .merch-card .con-button.outline {
   border: 2px solid var(--text-color);
   color: var(--text-color);
 }
+
+merch-card.special-offers del span[is="inline-price"] .price {
```
Comment (yesil) [inline libs/blocks/merch-card/merch-card.css:21]: small detail but we could apply this style to the whole `merch-card.special-offers del span[is="inline-price"] `

**Review by yesil — APPROVED** [review]

File: libs/blocks/merch-card/merch-card.css (hunk 1 above)
Comment (VKniaz) [reply libs/blocks/merch-card/merch-card.css]: updatd

**Comment by yesil** [comment]
@VKniaz could you please update your test urls and provide valid milo urls instead of cc ones?

**Comment by VKniaz** [comment]
@yesil done.

**Pull request merged**